### PR TITLE
feat: switchboard for active device tracking

### PR DESCRIPTION
**Pull request by ChewingGlass**

### CODE REVIEW

File: Anchor.toml [hunk 1]
```diff
@@ -58,6 +52,24 @@ program = "./deps/solana-program-library/account-compression/target/deploy/spl_n
...
+url = "https://api.mainnet-beta.solana.com"
+
+[[test.validator.clone]]
+address = "JBu1AL4obBcCMqKBBxhpWCNUt136ijcuMZLFvTP7iWdB"
+
+
+[[test.validator.clone]]
+address = "SW1TCH7qEPTdLsDHRgPuMQjbQxKdH2aBStViMFnt64f"
+
+[[test.validator.clone]]
```
Comment (ThornM9) [inline Anchor.toml:65]: Add a comment on these clones saying what they're for

File: packages/helium-cli/src/bootstrap.ts [hunk 2]
```diff
@@ -1,481 +0,0 @@
-import {
```
Comment (ThornM9) [inline packages/helium-cli/src/bootstrap.ts:1]: Why's this file being deleted?

File: packages/helium-cli/src/utils.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+import fetch from "node-fetch";
+
+export async function exists(
+  connection: Connection,
+  account: PublicKey
+): Promise<boolean> {
+  return Boolean(await connection.getAccountInfo(account));
+}
+
+export async function createAndMint({
```
Comment (ThornM9) [inline packages/helium-cli/src/utils.ts:27]: This function is duplicating a lot of the logic that already exists in spl-utils

**Review by ThornM9 — CHANGES_REQUESTED** [review]

File: packages/helium-cli/src/bootstrap.ts (hunk 2 above)
Comment (ChewingGlass) [reply packages/helium-cli/src/bootstrap.ts:1]: Replaced with create-dao and create-subdao

File: packages/helium-cli/src/utils.ts (hunk 3 above)
Comment (ChewingGlass) [reply packages/helium-cli/src/utils.ts:27]: Not duplicating, it's using the functions from spl utils.

**Review by ThornM9 — APPROVED** [review]

**Pull request merged**